index.html

```diff
@@ -701,5 +605,7 @@ <h1>Analytics</h1>
     integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
   <script src="js/data.js"></script>
   <script src="js/main.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/lodash@4.17.21/lodash.min.js"></script>
+  <script src="lodash.js"></script>
```

Comment:
- Don't forget to include library scripts _before_ your custom scripts (`data.js` &`main.js`).
- You probably don't need two copies of Lodash.